OpenMarket/OpenMarket/Resource/Controller/ProductsViewController.swift

```diff
@@ -0,0 +1,207 @@
+//
+//  OpenMarket - ProductsViewController.swift
+//  Created by yagom. 
+//  Copyright © yagom. All rights reserved.
+// 
+
+import UIKit
+
+// MARK: ProductsViewController
+final class ProductsViewController: UIViewController {
+    enum Constant {
+        static let edgeInsetValue: CGFloat = 8
+    }
+
+    private let productResponseNetworkManager = NetworkManager<ProductListResponse>()
+    private var currentPage: Int = 1
+    private var segmentIndex = 0
+
+    private var productsData: ProductListResponse? {
+        didSet {
+            DispatchQueue.main.async {
+                self.collectionView.reloadData()
+            }
+        }
+    }
+
+    private lazy var segment: UISegmentedControl = {
```

Comment:
```suggestion
    private lazy var layoutSegmentedControl: UISegmentedControl = {
```

UI요소들은 자신이 어떤 타입인지 네이밍에 나타내주는 것이 좋습니다 ㅎㅎ(사용처에서 오해할 여지가 없고, 직관적임)

ex, titleLabel, commentListCollectionView, cancelButton

Reply:
LayoutType 생성하면서 변경했습니다.!!!!!!!!